api/server.py

```diff
@@ -50,4 +50,5 @@ def stampify_url():
     try:
         return _stampifier.stampify().stamp_html
     except Error as err:
-        return err.message
+        return render_template('error_screen.html',
```

Comment:
Please this pylint error
`api/server.py:8:0: E0611: No name 'Error' in module 'error' (no-name-in-module)`

Reply:
It worked for me. :/
`$ pylint api/

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
`